crates/db_schema/src/source/custom_emoji_keyword.rs

```diff
@@ -0,0 +1,25 @@
+#[cfg(feature = "full")]
+use crate::schema::custom_emoji_keyword;
+use serde::{Deserialize, Serialize};
+use typed_builder::TypedBuilder;
+
+#[derive(PartialEq, Eq, Debug, Clone, Serialize, Deserialize)]
+#[cfg_attr(feature = "full", derive(Queryable, Associations, Identifiable))]
+#[cfg_attr(feature = "full", diesel(table_name = custom_emoji_keyword))]
+#[cfg_attr(
+  feature = "full",
+  diesel(belongs_to(crate::source::custom_emoji::CustomEmoji))
+)]
+pub struct CustomEmojiKeyword {
+  pub id: i32,
+  pub custom_emoji_id: i32,
+  pub keyword: String,
+}
+
+#[derive(Debug, Clone, TypedBuilder)]
+#[cfg_attr(feature = "full", derive(Insertable, AsChangeset))]
+#[cfg_attr(feature = "full", diesel(table_name = custom_emoji_keyword))]
+pub struct CustomEmojiKeywordInsertForm {
+  pub custom_emoji_id: i32,
```

Comment:
You should use newtype pattern for these as they are in a lot of places. That way the Rust typechecker can ensure that someone doesnt pass an id from a different table.

Reply:
Fixed